config/crd/bases/vm.neon.tech_virtualmachines.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
this file generated (by `make generate`) so it will be rewritten.

Reply:
@sharnoff 

and overall question - why need downgrade neonvm code ? I running current code over k8s 1.23 version (kind) and it works well. I know there are some issues with scheduler plugin but that plugin is not part of neonvm codebase. Or I'm wrong ?

Reply:
> this file generated (by make generate) so it will be rewritten

This is the rewritten version — downgrading the generator was required.

> why need downgrade neonvm code? [ ... ] I know there are some issues with scheduler plugin but that plugin is not part of neonvm codebase.

You're correct. Unfortunately, the scheduler plugin will use > 1.23 if any of its dependencies (including neonvm) use > 1.23.